src/SablierV2LockupDynamic.sol

```diff
@@ -186,6 +176,18 @@ contract SablierV2LockupDynamic is
     //////////////////////////////////////////////////////////////////////////*/
 
     /// @inheritdoc SablierV2Lockup
+    /// @dev The streaming function is:
```

Comment:
Should we add notice tag here as well? Half information is in `SablierV2Lockup` and half here.

@notice Calculates the streamed amount of the stream without looking up the stream's status. This function is declared in `SablierV2Lockup`.

Reply:
we have `/// @inheritdoc SablierV2Lockup` and the `override` specifier, so i think that is enough

Reply:
Yes I agree and precisely the reason why I pointed that out because the reader has to go to the abstract just to read the notice tag.

Reply:
>because the reader has to go to the abstract just to read the notice tag.

that's pretty much the same for every function that uses inheritdoc to an interfaces, e.g. `createWithDurations` `createWithTimestamps`

Reply:
OK. I agree.